script/make_utils/check_internal_links.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# We don't want links to the main branch, even if it's public. Instead, release branch should be
+# targeted
+if grep -r "tree/main" docs | grep "\.md:" | grep -v "https://huggingface.co/spaces/"; then
+    echo -n -e "\nThe above links contain references to the main banch. Please remove them as only "
+    echo "release branches should be referenced."
+    exit 255
+fi
+
+
+# We don't want links to our internal repositories (COncrete ML or Concrete), expect if they are
```

Comment:
Small typo "COncrete ML" instead of "Concrete ML"

Reply:
indeed, thanks